### PR TITLE
Modify upstream for auto build

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM  rhel7:7-released
+LABEL maintainer="OpenShift Development <dev@lists.openshift.redhat.com>" \
+      com.redhat.component="golang-github-prometheus-alertmanager-docker" \
+      name="openshift3/prometheus-alertmanager" \
+      version="v0.15.0-rc3" \
```
Comment (simonpasquier) [inline Dockerfile.rhel7]: To be changed once 0.15.0 is officially out.

File: golang-github-prometheus-alertmanager.spec [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+%global debug_package   %{nil}
+
+%global provider        github
+%global provider_tld    com
+%global project         prometheus
+%global repo            alertmanager
+# https://github.com/prometheus/alertmanager
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          5e86f61bd73c6325d6049ab3dbcb468ede26dfe0
```
Comment (simonpasquier) [inline golang-github-prometheus-alertmanager.spec]: Not sure what to put here.

File: golang-github-prometheus-alertmanager.spec [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+%global gopathdir       %{_sourcedir}/go
+%global upstream_ver    0.15.0-rc3
```
Comment (simonpasquier) [inline golang-github-prometheus-alertmanager.spec]: To be changed once 0.15.0 is officially out.

**Comment by pgier** [comment]
lgtm

**Pull request merged**